lib/aramanim.py

```diff
@@ -4,46 +4,15 @@
 from manim import FadeIn, FadeOut, Animation, Scene
 from manim import always_redraw
 import numpy as np
-from numpy import linalg as la
 from objects import Scissors, Stopwatch
 
-# np վեկտորների համար ՛մեթոդներ՛
-def length(array):
-    return la.norm(array)
-
-def normalized_vector(array):
-    array = np.array(array)
-    return (1/length(array)) * array
-
-def rotate2angle(array, alpha):
-    array = np.array(array)
-    rotation__matrix = [
-        [np.cos(alpha), -np.sin(alpha), 0],
-        [np.sin(alpha), np.cos(alpha), 0],
-        [0, 0, 1],
-    ]
-    return np.matmul(rotation__matrix, array)
-
-def normal(array):
-    array = np.array(array)
-    return normalized_vector(rotate2angle(array, PI/2))
-
-# հայերեն լեզվով LaTeX աշխատացնելու համար
-XELATEX_preamble = r"""
-\usepackage{tikz}
-\usepackage{fontspec}
-\setmainfont{DejaVu Serif}
-\usepackage{amsfonts,amssymb,amsthm,mathtools}
-\usepackage{amsmath}
-\usepackage{upgreek}
-\usepackage{mathrsfs}
-"""
-XELATEX = TexTemplate('xelatex', '.pdf', preamble=XELATEX_preamble)
```

Comment:
It seems this piece of code is being deleted here and is not duplicated anywhere. But I would like it to remain in some form. I think over time we will use the XeLaTeX compiler

Reply:
Can you look at `lib/constants.py` file, do you think we should copy this to that file? I have defined all the tex related objects there.

Reply:
Yes, it will be great